cni/network/network.go

```diff
@@ -1468,3 +1491,19 @@ func convertCniResultToInterfaceInfo(result *cniTypesCurr.Result) network.Interf
 
 	return interfaceInfo
 }
+
+func (plugin *NetPlugin) validateArgs(args *cniSkel.CmdArgs, nwCfg *cni.NetworkConfig) error {
+	if !isValidString(args.ContainerID) || !isValidString(args.IfName) {
+		return errors.New("invalid args value")
+	}
+	if !isValidString(nwCfg.Bridge) {
+		return errors.New("invalid network config value")
+	}
+
+	return nil
+}
+
+// returns true if the string fully consists of zero or more alphanumeric, dots, dashes, parentheses, or underscores
+func isValidString(value string) bool {
+	return validRegexp.MatchString(value)
+}
```

Comment:
move the doc to the regexp declaration, rename it to something like `allowedInput` and drop this extra function def entirely